src/auth/googleauth.ts

```diff
@@ -583,6 +591,8 @@ export class GoogleAuth {
     options = options || {};
     const client = new JWT(options);
     client.fromAPIKey(apiKey);
+    client.defaultServicePath = this.defaultServicePath;
```

Comment:
I think this is a special edge case where we don't need to set `useJWTAccessAlways`, where we're providing an explicit API key to use (which I'm betting is why we don't set defaultServicePath either).